ampl-converter/src/main/java/com/powsybl/ampl/converter/AmplNetworkWriter.java

```diff
@@ -1580,7 +1589,7 @@ private void writeGenerators(AmplExportContext context) throws IOException {
                         .writeCell(g.getReactiveLimits().getMaxQ(0))
                         .writeCell(g.getReactiveLimits().getMaxQ(minP))
                         .writeCell(g.isVoltageRegulatorOn())
-                        .writeCell(g.getTargetV() / vb)
```

Comment:
It is quite suprising that we don't have to do a fix for ratio tap changers indeed...